clang/test/Driver/ps5-linker.c

```diff
@@ -46,3 +46,26 @@
 
 // CHECK-SYSROOT: {{ld(\.exe)?}}"
 // CHECK-SYSROOT-SAME: "--sysroot=mysdk"
+
+// Test that "." is always added to library search paths. This is long-standing
+// behavior, unique to PlayStation toolchains.
+
+// RUN: %clang --target=x64_64-sie-ps5 %s -### 2>&1 | FileCheck --check-prefixes=CHECK-LDOT %s
+
+// CHECK-LDOT: {{ld(\.exe)?}}"
+// CHECK-LDOT-SAME: "-L."
+
+// Test that <sdk-root>/target/lib is added to library search paths, if it
+// exists and no --sysroot is specified.
```

Comment:
Demonstrate that --sysroot causes no `target/lib` to be appended? Or is that tested elsewhere?

Reply:
This is what the RUN on line 63 does ... I hope! (Prior to 1a6dc7d, this was on line 68).

Reply:
Ah missed that, sorry!